contents/docs/data-warehouse/sources/index.mdx

```diff
@@ -2,10 +2,22 @@
 title: Linking sources for PostHog's data warehouse
 ---
 
-import DWInstallationPlatforms from '../_snippets/dw-installation-platforms'
+import { ManagedSources, SelfManagedSources } from '../_snippets/dw-installation-platforms'
 
-Beyond the [PostHog data](/docs/data-warehouse/sources/posthog) already in your data warehouse, you can link and query external sources too. To get started with  linking an external source to PostHog's data warehouse, choose your source below.
+## PostHog data
 
-<DWInstallationPlatforms />
+Much of your PostHog data is available in the data warehouse by default. This includes data like events, persons, sessions, groups, and the query log. Learn how to [use PostHog as your data source](/docs/data-warehouse/sources/posthog).
 
-You can also learn more about sources in the [data pipeline docs](/docs/cdp/sources).
\ No newline at end of file
+## Managed sources
```

Comment:
Is this not the same set of sources as `contents/docs/cdp/_snippets/cdp-sources.tsx`?